packages/terra-popup/src/_PopupContent.jsx

```diff
@@ -223,12 +227,18 @@ class PopupContent extends React.Component {
 
     const heightData = isHeightAutomatic ? { 'data-terra-popup-automatic-height': true } : {};
     const widthData = isWidthAutomatic ? { 'data-terra-popup-automatic-width': true } : {};
+    let consumerAriaLabel = '';
+    const modifiedCustomProps = { ...customProps };
+    if (modifiedCustomProps && modifiedCustomProps.ariaLabel) {
+      consumerAriaLabel = modifiedCustomProps.ariaLabel;
+      delete modifiedCustomProps.ariaLabel;
+    }
```

Comment:
```suggestion
```
Same here I think we can remove this as well

Reply:
Reverted as suggested.